DDDigi/include/DDDigi/DigiAttenuator.h

```diff
@@ -44,7 +45,7 @@ namespace dd4hep {
       /// Default destructor
       virtual ~DigiAttenuator();
       /// Callback to read event attenuator
-      virtual double operator()(const DigiCellData& data)  const  override {
```

Comment:
clang complained: error: 'operator()' marked 'override' but does not override any member functions